README.md

```diff
@@ -55,22 +56,31 @@ The app is hosted on heroku, you can view it [here](http://nazareth-events.herok
     * main.CSS
   * images/  
   * js/
+    * helpers/
```

Comment:
do we even need a file structure ?
I'm not sure anyone who wants to know about the app would care

Reply:
Extra but helps developers navigate